### PR TITLE
Python Integration to send raw attacks by cron 

### PR DESCRIPTION
**Pull request by 416e64726579**
Added python integration which fetches raw attacks through Wallarm API and send them to an http collector of Loggly.
The code repo is here https://github.com/416e64726579/Wallarm-API-Client

### CODE REVIEW

File: terraform/system-apps/log-integration/log-integration/.helmignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```
Comment (vgartvich-wallarm) [inline terraform/system-apps/log-integration/log-integration/.helmignore]: Please move the component to new directory terraform/integration-demos/ (please update manage-env.sh as well)

File: terraform/system-apps/log-integration/main.tf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  set_string {
+    name  = "wallarm_api"
+    value = "us1.api.wallarm.com"
+  }
+
+  set_string {
+    name  = "image"
+    value = "awallarm/log-exporter:latest"
```
Comment (vgartvich-wallarm) [inline terraform/system-apps/log-integration/main.tf]: Please point to the new repo.

File: terraform/system-apps/log-integration/main.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    name  = "pool_id"
+    value = 9
+  }
+
+  set_string {
+    name  = "schedule"
+    value = "0 * * * *"
+  }
+
+  set_string {
```
Comment (vgartvich-wallarm) [inline terraform/system-apps/log-integration/main.tf]: Please use the existing variable.

File: terraform/system-apps/log-integration/main.tf [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    name  = "image"
+    value = "awallarm/log-exporter:latest"
+  }
+
+  set_sensitive {
+    name  = "wallarm_uuid"
+    value = var.loggly_wallarm_uuid
```
Comment (vgartvich-wallarm) [inline terraform/system-apps/log-integration/main.tf]: Rename the variable to wallarm_api_uuid.

File: terraform/system-apps/log-integration/main.tf [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  set_sensitive {
+    name  = "wallarm_secret"
+    value = var.loggly_wallarm_secret
```
Comment (vgartvich-wallarm) [inline terraform/system-apps/log-integration/main.tf]: Rename the variable to wallarm_api_secret.

**Review by vgartvich-wallarm — CHANGES_REQUESTED** [review]

**Comment by 416e64726579** [comment]
Please, take a glance at the latest commit. I've fixed all discussed.

**Pull request merged**